packages/backend/src/factcollectors/store/branchcountextractorsstore.ts

```diff
@@ -7,14 +7,30 @@ import {
 } from '@spotify/backstage-plugin-soundcheck-common';
 import { BranchCountFactCollectorSchema } from './utils';
```

Comment:
This should be changed to:
`import { BranchesDetailsFactCollectorSchema } from './utils';`